addons/account_check_printing/__openerp__.py

```diff
@@ -10,7 +10,10 @@
 The check settings are located in the accounting journals configuration page.
     """,
     'website': 'https://www.odoo.com/page/accounting',
-    'depends' : ['account'],
+    'depends' : [
+        'account',
+        'account_voucher',  # Not for normal Odoo, but needed for migration
```

Comment:
Does this forces the installation when you do `--update=all`? Maybe you need to update manually in base module the state to `to install` for this module when `account_check_printing` is installed